### ISSUE TITLE
Cannot highlight text in slot (Vue3)

### ISSUE THREAD

**Issue by nterray** [kawamataryo/vue-word-highlighter#469]
Hello,

In a Vue3 app, text passed through a `<slot>` is not highlighted nor rendered. Please note that it worked fine with Vue2.
```typescript
<template>
  <div>
    <h1>Test slots</h1>
    <WordHighlighter :query="'ipsum'">Lorem ipsum doloret</WordHighlighter>
    <hr>
    <HighlightItem>Lorem ipsum doloret</HighlightItem>
  </div>
</template>

<script>
import WordHighlighter from "vue-word-highlighter";
import HighlightItem from "./HighlightItem.vue";
import { defineComponent } from "vue";

export default defineComponent({
  name: "App",
  components: {
    WordHighlighter,
    HighlightItem,
  },
});
</script>
```

With `HighlightItem.vue`:
```typescript
<template>
  <div>
    HighlightItem: 
    <WordHighlighter :query="'ipsum'"><slot></slot></WordHighlighter>
  </div>
</template>
<script setup lang="ts">
import WordHighlighter from "vue-word-highlighter";
import { defineComponent } from "vue";

export default defineComponent({
  name: "HighlightItem",
  components: {
    WordHighlighter,
  },
});
</script>
```

See the codesandbox:
https://codesandbox.io/s/issue-with-slots-29i9w5?file=/src/App.vue

Is it an expected behavior or a regression?

Best regards,
Nicolas

**Comment by kawamataryo** [comment]
Thank you for the Issue.
I'll check it.

**Comment by kawamataryo** [comment]
@nterray 
There was a mistake in the process of getting the text for the Vue3 slot.
A corrected version will be released soon.

**Comment by kawamataryo** [comment]
Fixed in v1.0.6.

See the codesandbox:
https://codesandbox.io/s/issue-with-slots-forked-ixrdxo
